scalding-date/src/main/scala/com/twitter/scalding/DateOps.scala

```diff
@@ -51,13 +47,15 @@ object DateOps extends java.io.Serializable {
   /**
    * Return the guessed format for this datestring
    */
-  def getFormat(s: String): Option[String] =
-    DATE_FORMAT_VALIDATORS.find { _._2.findFirstIn(prepare(s)).isDefined }.map(_._1)
+  def getFormat(s: String): Option[Format] =
```

Comment:
Looks like this is a public method. So maybe it would be a better idea to add a new method of signature `String -> Option[Format]` rather than changing the return type of this method?